### PR TITLE
Adding This week in Rust Fest No 6.

### PR DESCRIPTION
**Pull request by gnunicorn**
Venue, Sponsors, Speakers and a party parrot

Screenshot:
screen shot 2016-07-15 at 13 21 27-fullpage

### CODE REVIEW

File: _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Aside from [Microsoft Berlin](https://www.microsoft-berlin.de/de-de/berlin/), two other organisations decided to join the list of sponsors of RustFest this week. The always awesome [Mozillians](https://www.mozilla.org/en-US/) and [QiTASC](http://www.qitasc.at/), a Quality Assurance Provider for mobile from Vienna, decided to join the ranks. The website has been updated accordingly.
+
+![New Sponsors](/assets/posts/twirf-6/new-sponsors.png)
+
+Welcome aboard, and thank you very much!
+
+
+## First CfP speakers confirmed
+
+As promised last week already, we've also selected the first few speaker from the CfP and informed them about our decision. And we are very happy to hear back from so many already, that they can indeed make it. We'll share the first few names and talk descriptions starting next week, but I can already tell you that you'll see some familiar faces as well as complete new comers to public speaking with topics ranging from hardware, testing over concurrency and disassemblers all the way to community, learning and empathy. We are very happy with the set of talks and can't wait to inform you about them.
```
Comment (badboy) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: new comers → newcomers ?

File: _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "This Week in RustFest 5: Announcing the venue, more awesome sponsors and first CfP speaker confirmed"
```
Comment (johannhof) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: I think you mean 6

Comment (johannhof) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: also `and first CfP speakers confirmed`

Comment (johannhof) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: and the `confirmed` at the end looks unnecessary

File: _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+socialImageSrc: '/assets/posts/twirf-6/new-sponsors.png'
+---
+
+Oh, what a great week for RustFest!
+
+## Venue, in the center of Berlin
+
+Earlier this week really great news arrived us: **our most anticipated venue confirmed they will host RustFest 2016!**
```
Comment (johannhof) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: is "arrived us" a thing? maybe just "arrived" :)

File: _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+![RustFest Venue 2016](/assets/posts/twirf-6/venue-house.png)
+
+
+We got the pleasure to visit the "Atrium", their newly equipped conference setting, a few weeks ago and directly knew: this is where we want to host RustFest. I mean, just look at it. And that picture doesn't even cover all the amazing space we're gonna have. Check out [their website to get some impressions](https://www.microsoft-berlin.de/de-de/berlin/).
+
+
+![RustFest Venue 2016](/assets/posts/twirf-6/venue-room.png)
+
+
+But wait, there's more. Microsoft Berlin was so impressed with RustFest and what we are up to, they decided to sponsor the event. WHAAA? Yes, Microsoft Berlin let's use the venue _for both days_ free of charge! Our dreams come true.
```
Comment (johannhof) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: let's -> lets us

File: _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+You know what that means? Well, for once, **WE WILL HAVE WORKSHOPS**. With the venue being sponsored by awesome Microsoft, we have at least one awesome workshop room right on side. Further more we will convert the big room we have for talks on the first day to a big meet-n-greet-n-hack area during the second. So for everyone, who's just happy to finally meet other community members in person and just want to hang out and chat, we'll have space for that, too. Oh, my. This is shaping up to be quite awesome!
+
+
+## More sponsors
+
+Aside from [Microsoft Berlin](https://www.microsoft-berlin.de/de-de/berlin/), two other organisations decided to join the list of sponsors of RustFest this week. The always awesome [Mozillians](https://www.mozilla.org/en-US/) and [QiTASC](http://www.qitasc.at/), a Quality Assurance Provider for mobile from Vienna, decided to join the ranks. The website has been updated accordingly.
```
Comment (johannhof) [inline _posts/2016-07-15-this-week-in-rustfest-6-venue-sponsors-and-the-speakers.md]: Before seeing the link target I wondered who you refer to as "Mozillians". I think I'd rather go with "Mozilla" instead. The sentence would still work I think. :)

**Comment by gnunicorn** [comment]
All fixed @johannhof , @badboy . I'm off now, but feel free to merge at any time.

**Pull request merged**